tests/spec/fs.unlink.spec.js

```diff
@@ -104,3 +104,22 @@ describe('fs.unlink', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+describe('fsPromises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+  it('should return an error if trying to delete a file that does not exist', function(done) {
+    var fsPromises = util.fs().promises;
+
+    fsPromises.unlink('/myFile')
+      .then(result => expect(result).not.to.exist)
```

Comment:
This line is also unneeded in a promise-based test, so please remove this line.

Reply:
@humphd Would you please help me on this issue? Do I need to include it?

Reply:
I think you can remove it, yes.  It's not wrong to have it, but it's not doing anything we care about.  With callbacks, you need to do this, because the callback gets called in both the error and success cases, so you have to examine state for both.  With Promises, only one of the two conditions will happen.